### ISSUE TITLE
Danger Zone FPS

### ISSUE THREAD

**Issue by randomuser123123** [iseekwonderful/csgoGlow#264]
Kind of has nothing to do with the hack but i was hoping someone could help... Basically my fps dips to even 20 while playing danger zone and was wondering if anyone else has had this issue? Just looking for an FPS boost really.

**Comment by ethano8225** [comment]
yep its a big map not really gonna be ideal for macs just turn down settings
